qemu/tests/timer_rtc_sync.py

```diff
@@ -77,6 +77,13 @@ def verify_timedrift(session, is_hardware=False):
                       " host's system time. Host time: '%s', guest time:"
                       " '%s'" % (time_type, host_time, guest_time))
 
+    def disable_local_rtc(session=None):
+        cmd_func = session.cmd_output if session else process.getoutput
+        if "RTC in local TZ: yes" in cmd_func("timedatectl status"):
+            cmd_func("timedatectl set-local-rtc 0")
+            if session and "NTP service: active" in cmd_func("timedatectl"):
+                vm.reboot()
```

Comment:
1. Since `session` is available, suggest `vm.reboot(session)` to use it directly instead of get it in the reboot function again
2. We need a new session after reboot,  to be used in the follow context. such as L111